src/errors/runtime/index.js

```diff
@@ -9,7 +9,18 @@ import util from 'util';
 import semver from 'semver';
 import { removePreventModuleCachingSuffix } from '../test-run/utils';
 
-const ERROR_SEPARATOR = '\n\n';
+const ERROR_SEPARATOR        = '\n\n';
+const NO_STACK_AVAILABLE_MSG = 'No stack trace available for this error';
+const MODULE_NOT_FOUND_CODE  = 'MODULE_NOT_FOUND';
+const REPORTER_MODULE_PREFIX = 'testcafe-reporter-';
```

Comment:
Move to the separate `/report/prefix.ts` file and reuse here and [here](https://github.com/DevExpress/testcafe/pull/7759/files#diff-ae666665181ee43a04b6e756d3f8fbd260352222e8942b31d0ec501187b57d30R5).